lib/killer.js

```diff
@@ -21,10 +21,10 @@ module.exports = function kill(child) {
     } else {
       if (hasPS) {
         psTree(child.pid, function(err, kids) {
-          spawn('kill', ['-s', KILL_SIGNAL, child.pid].concat( kids.map(p => p.PID) )).on('close', resolve);
+          spawn('kill', [KILL_SIGNAL, child.pid].concat( kids.map(p => p.PID) )).on('close', resolve);
         });
       } else {
-        exec('kill -s ' + KILL_SIGNAL + ' ' + child.pid, resolve);
+        exec('kill ' + KILL_SIGNAL + ' ' + child.pid, resolve);
```

Comment:
also here

Reply:
Will do, I wasn't sure which way around you wanted it. Will fix up and update the PR.